packages/core/src/App/Containers/PasswordSelectionModal/password-selection-modal.jsx

```diff
@@ -3,10 +3,9 @@ import classNames from 'classnames';
 import { Field } from 'formik';
 
 import { Button, PasswordInput, PasswordMeter, Text } from '@deriv/components';
-import { getErrorMessages, redirectToSignUp } from '@deriv/shared';
+import cacheTrackEvents, { getErrorMessages, redirectToSignUp } from '@deriv/shared';
```

Comment:
should be `import { getErrorMessages, redirectToSignUp, cacheTrackEvents } from '@deriv/shared';`